FireRender.Maya.Src/FireRenderObjects.cpp

```diff
@@ -152,10 +152,13 @@ HashValue FireRenderNode::CalculateHash()
 {
 	HashValue hash = FireRenderObject::CalculateHash();
 	auto dagPath = DagPath();
+
+	MString name = dagPath.fullPathName();
+
 	if (dagPath.isValid())
 	{
 		hash << dagPath.isVisible();
-		hash << dagPath.inclusiveMatrix();
+		hash << 0; // dagPath.inclusiveMatrix();
```

Comment:
Looks like unnecessary line of code

Reply:
If inclusive matrix call is about to be removed isVisible call is also unnecessary